website/docs/docs/build/measures.md

```diff
@@ -56,8 +56,32 @@ Supported aggregations include:
 | sum_boolean       | A sum for a boolean type |
 | count_distinct    | Distinct count of values |
 | median           | Median (p50) calculation across the values |
-| percentile        | Percentile calculation across the values  |
+| percentile        | Percentile calculation across the values. |
 
+**Percentile aggregation example**<br />
```

Comment:
sugg: promote to level H4 heading  since it already reads and looks like a heading and also makes the example linkable/shareable

```suggestion
#### Percentile aggregation example
```